lib/compilers/win32.js

```diff
@@ -22,41 +22,41 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
```

Comment:
Perhaps a "use strict" is still worthwhile in the code here? I see other files have it.

Reply:
Yeah, probably, I'll add it in. The unfortunate thing about it being non-default is you have to remember to put it there!